r-pkg/uptasticsearch/tests/testthat/test-repo_characteristics.R

```diff
@@ -39,7 +39,7 @@ test_that("R CMD check should not return any errors", {
 })
 
 test_that("R CMD check should not return any warnings", {
-    expect_true(length(x[["warnings"]]) == 0)
+    expect_true(length(x[["warnings"]]) == 1)
```

Comment:
Can you please add a comment showing the 1 warning we are tolerating and explaining the conditions under which it can arise (i.e. on a system without pdflatex)?

Also would like @austin3dickey 's opinion on this